libstorage/src/Disk.cc

```diff
@@ -2113,6 +2113,17 @@ int Disk::doRemove( Volume* v )
 	    getStorage()->removeDmMapsTo( getPartDevice(p->OrigNr()) );
 	    ret = v->prepareRemove();
 	    }
+        // before deleting partitions ensure that efi do not contain it
+	if( ret==0 && !p->created() )
+            {
+	    std::ostringstream cmd_line;
+            cmd_line.str("");
```

Comment:
This is useless, see http://www.cplusplus.com/reference/sstream/ostringstream/ostringstream/.